compute_tools/src/bin/compute_ctl.rs

```diff
@@ -116,17 +116,17 @@ fn main() -> Result<()> {
     let pageserver_connstr = spec
         .cluster
         .settings
-        .find("zenith.page_server_connstring")
+        .find("neon.pageserver_connstring")
         .expect("pageserver connstr should be provided");
     let tenant = spec
         .cluster
         .settings
-        .find("zenith.zenith_tenant")
+        .find("neon.tenantid")
```

Comment:
NIT: A bit odd to have those not in `tenant_id`/`timeline_id` form.

Reply:
+1